### PR TITLE
Bump netutils to 0.2.1

### PR DESCRIPTION
**Pull request by jdrew82**
Netutils was recently bumped to 0.2.1 with some bug fixes and ASA parser addition. We need to bump it here so other apps can use the parser and bug fixes.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ nornir-napalm = "^0.1.1"
 nornir-jinja2 = "^0.1.1"
 nornir-netmiko = "^0.1.1"
 pynautobot = "^1.0.1"
-netutils = "^0.1.2"
+netutils = "^0.2.1"
```
Comment (itdependsnetworks) [inline pyproject.toml:29]: I think I may get confused by poetry's syntax sometimes, is this better served with a `~0.X` syntax? My intention of the original was it should have just picked up for any user installing and this should not have been needed, but I suspect that is not the case. If so, should probably fix the original issue.

Comment (jvanderaa) [reply pyproject.toml:29]: If just any version should be installed, based on the time of install then `*` would handle this.

Comment (itdependsnetworks) [reply pyproject.toml:29]: It is not any for sure, it is above 1.2 that they would need, but I am not sure what that means for poetry spec.

[the remark this answers is not in the document]
Comment (itdependsnetworks) [reply pyproject.toml:29]: ok, that kinda makes sense. If that agreed, then that is good to go.

[the remark this answers is not in the document]
Comment (jdrew82) [reply pyproject.toml]: Works for me. I've pushed a commit with this change.

Comment (jdrew82) [reply pyproject.toml]: Updated to use ^0 per the discussion below.

**Review by jvanderaa — APPROVED** [review]
Looks good at this point.

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**